src/Stripe.net/Services/PaymentMethods/PaymentMethodBancontactOptions.cs

```diff
@@ -0,0 +1,6 @@
+namespace Stripe
```

Comment:
it's weird it doesn't see this one as a rename

Reply:
yeah I think between the class name changing, and removing the unneeded json using statement, it's different enough to git to think it's a new file 🤷